download-files.sh

```diff
@@ -3,16 +3,21 @@ set -e
 
 source .env
 
-echo "This script will download a few GB of Fedora and a few MB of Kernel into the current directory."
-echo "Look inside the script if you would rather download manually."
+echo "This script will download the Kernel and ca. 900Mb of Fedora raw file into the current directory."
 echo
 read -p "Continue? [y/N] " -n 1 -r
 echo
 if [[ $REPLY =~ ^[Yy]$ ]]
 then
     # Download kernel
-    wget https://xff.cz/kernels/5.7/pp1.tar.gz -O pp.tar.gz
+    echo "We need to download kernel only for the uboot.bin file"
+    echo "Kernel will be installed from repos"
+    wget https://xff.cz/kernels/5.8/pp2.tar.gz -O pp.tar.gz
```

Comment:
Is this going to be a problem for those running a Braveheart image? (pp1)

Reply:
It could. If I remember there was a problem with ArchLinuxARM on BH phones. It's better to take pp1.tar.gz instead pp2